### ISSUE TITLE
This doesn't work on next.js

### ISSUE THREAD

**Issue by techandmedia** [Leonard-Li777/antd-table-infinity#16]
Using the latest version of React, React-DOM and nextjs
"react": "^16.9.0",
"react-dom": "^16.9.0",
"next": "^9.0.3",

Throwing an error: "ReferenceError: window is not defined"

Nextjs already provide a guideline of how to make it works:
https://github.com/zeit/next.js/wiki/FAQ

Not sure how to make it work though,

**Comment by ayushkhanduri** [comment]
yup , i also encountered that issue , the no-ssr plugin is not working , well the main reason is that it uses intersection-observer and the first thing that intersection observer does is that it checks if it's instance has been created and attached to the global object . It happens before the componentdidmount cycle for the component. Just importing that file causes the application to crash . I don't think that even the workarounds are worth it.

**Comment by thanglong134** [comment]
Is there any way to solve this problem?

**Comment by bconnorwhite** [comment]
SSR should be fixed with https://github.com/Leonard-Li777/antd-table-infinity/pull/28.

**Comment by tamert** [comment]
ReferenceError: window is not defined

**Comment by sagarj521** [comment]
this is still not working with nexjs is there any workaround for it?

**Comment by ManojShivaramAithal** [comment]
ReferenceError: window is not defined : Any Solution for this ?, Thanks in advance.
